src/test/java/sysc4806/graduateAdmissions/controller/RoleControllerTest.java

```diff
@@ -45,11 +45,11 @@
     @BeforeEach
     void setUpMocks() {
         Set<Privilege> studentPrivileges = new HashSet<>();
-        studentPrivileges.add(Privilege.builder().id(0L).owner(Owner.ALL_STUDENTS).build());
+        studentPrivileges.add(Privilege.builder().id(0L).owner(Owner.ALL_STUDENTS).target(Target.TERM).operation(Operation.DELETE).build());
         Set<Privilege> adminPrivileges = new HashSet<>();
-        studentPrivileges.add(Privilege.builder().id(1L).owner(Owner.SELF).build());
+        studentPrivileges.add(Privilege.builder().id(1L).owner(Owner.SELF).target(Target.USER).operation(Operation.UPDATE).build());
         Set<Privilege> professorPrivileges = new HashSet<>();
-        studentPrivileges.add(Privilege.builder().id(2L).owner(Owner.ALL_PROFS).build());
+        studentPrivileges.add(Privilege.builder().id(2L).owner(Owner.ALL_PROFS).target(Target.INTEREST).operation(Operation.CREATE).build());
 
```

Comment:
The mockito mocks probably shouldn't be in the the setup before every test because not all tests need to mock them. I shouldn't be telling a test to return a good result when it does something it shouldn't be doing.

Same thing with the privilege controller tests + theres no java doc in that class (I don't really care much about that though).